src/javascript/JContent/EditFrame/Box.scss

```
@@ -105,7 +105,7 @@
 
     padding: 0 var(--spacing-small);
 
-    background-color: var(--color-gray_light40);
+    background-color: var(--colorHeaderBackground);
```

can we provide default fallback in case style is reused and `--colorHeaderBackground` is not defined

```suggestion
    background-color: var(--colorHeaderBackground, var(--color-gray_light40));
```